### DISCUSSION TITLE
Images order incorrect ,if filenames contain multi number

### DISCUSSION THREAD

**Discussion by DTEX123** [ImageMagick/ImageMagick#7754] [Help] [answered]
### ImageMagick version

7.1.1.39

### Operating system, version and so on

windows10

### Description

![12](https://github.com/user-attachments/assets/5956cfef-5b56-4512-bb0f-07266d5ed4b7)
Here is the result
![spriteSheet](https://github.com/user-attachments/assets/7b0743da-6fae-4b41-a772-a34db4584507)
`magick montage -background none -tile x1 -geometry 338x332+0+0 SheetImages/*.png spriteSheet.png`

**Comment by dlemstra** [comment, marked as answer]
You will need to prefix `1/2/3` with a zero to resolve your issue. `image1.png` => `image01.png`.

**Comment by fmw42** [comment]
Images are normally read in alphabetic order not numeric.  As dlemstra said, put leading zeroes on your filename numbers.
